### DISCUSSION TITLE
Handling multimodal delay distributions

### DISCUSSION THREAD

**Discussion by adrian-lison** [epinowcast/epinowcast#104] [Ideas]
One main motivation for adding a non-parametric component to epinowcast in addition to the current parametric model is that delay distributions could be multimodal, and this is a problem for plain lognormal, gamma, exponential etc. delay models.

Just as an idea for discussion, could it be an alternative (in the sense of addition or replacement for non-parametric modeling) to offer support for mixtures of parametric distributions? For example, a typical delay distribution with two modes can be modeled as a mixture of a general lognormal (large weight) + a rather sharp lognormal with a larger mean (small weight).

When modeling this in stan, it could make sense to enforce that the second lognormal has the larger mean to avoid ambiguity.

Opinions (does this make sense, would it perform well, would it be efficient) welcome!

**Comment by seabbs** [comment]
Yeah, I really like this idea and I think it would be useful. An example use case would be where we know a delay is driven by two processes and we think both have distributions that differ.

I also think it might be fairly complicated to get it working. Probably need to do something with quite informative priors to enforce long/short means?

**Comment by adrian-lison** [comment]
This discussion is continued in our discourse community [here](https://community.epinowcast.org/t/handling-multimodal-delay-distributions/36).
